src/bicep-registry/scheduled-action/main.bicep

```diff
@@ -0,0 +1,138 @@
+// @resourceGroup @subscription @tenant
```

Comment:
This is a "scope directive". This is used to tell the build script what scopes are supported by this module.